public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
```

Comment:
What does this file do?

Reply:
The general purpose of the file is managing web crawler traffic to a web page. I believe this configuration tells web crawlers not to crawl any of the pages from the site served with this robots.txt file. This prevents descriptions of the site from being generated by search engine web crawlers, but does not prevent the search engine from indexing links to the site.